### ISSUE TITLE
feature request - option to change the menu button from "game exit" to interactive "quick menu"

### ISSUE THREAD

**Issue by ilmenit** [jimgraygit/Onion#397]
The stock OS or Bastard OS have the interactive quick menu available, while Onion base on key shortcuts to perform in-game operations. Maybe for some the shortcuts are faster and more natural but the interactive quick menu (even if not as default but as an option to turn on) would be a great addition. The most useful features of the menu would be for me:
- continue
- save state, the best if the screenshots could be seen.
- load state, the best if the screenshots could be seen.
- enter Retroarch menu
- restart the machine (useful considering load-last-save-on-start).
- exit game
I know that there's a Retroarch menu available and while it's great for changing options, it's not good for performing quick actions

**Comment by Olywa** [comment]
Hi there, while I appreciate it's not entirely what you are asking for, have you tried the Onion Launcher feature (it can be enabled in the APP section)? More info here: https://github.com/jimgraygit/Onion#onion-launcher

- Once enabled, you must reboot. The last game you played will be opened in 'Onion Launcher' mode. 
- To remain in the launcher you would press menu and then scroll through your game history inside the launcher (using dpad left/right) and then press menu again to select a chosen game from the launcher menu.
- Powering off will save the game before shutting down so you return to the same location on boot.
- If you exit the launcher it will start again after reboot -or- by going to App->'Start Launcher' (note, this is not the same as App->'Onion Launcher - Enabled/Disabled'

In either case I suggest to keep your request open so the developers may consider it.

**Comment by ilmenit** [comment]
> have you tried the Onion Launcher feature (it can be enabled in the APP section
yes, I did. The only thing I like about the Onion Launcher is the returning to the game on boot. The rest of it is not a good user experience for me. 
1. Pressing the menu button you get a Launcher's Quick Menu, with the main option to select games from the history. Are there really people for who, when playing a game, the most expected option is to select a previously played other game? I'd say working with the game states is much more common.
2. The Launcher's Quick Menu has these small icons (Menu, Start, X, Y) of quick options. The quick options however are are kind of pointless. Maybe there could be Load/Save state, but there is Game Play Time and Removing from History instead. Is really Game Play Time that important for regular activity?
3. Launcher has ton of keyboard shortcuts that user MUST remember but I cannot (maybe because I don't play the console regularly). The point when I disabled Launcher was when instead of Save State I clicked Load State shortcut and lost progress (again). 
4. Maybe I don't see an option but how to select other state? Only going to Retro Arch menu?
5. Exiting Launcher and returning to Main Menu to select some other game... disables Launcher. It's not there when other game is selected. It's very counter-intuitive. 
Considering all these Onion for me is much better with the Launcher disabled, but then Quick Menu button similar to Stock or Bastard OS would be useful.
